runtime/compiler/env/j9method.cpp

```diff
@@ -8176,7 +8176,7 @@ TR_J9ByteCodeIlGenerator::runFEMacro(TR::SymbolReference *symRef)
                   switch (argType[0])
                      {
                      case 'L':
-                     case '[':
+                     case '[': //Same question as above: not sure if for 'Q' should we have something like sourceType = "Qjava/lang/Object"? 
```

Comment:
As above, I would just add a `case 'Q':` for now, and have that map to `L` and `extra_L`